### PR TITLE
Simple grid #260

### PR DESCRIPTION
**Pull request by metalerk**
# Pull request for #260 

## Changes

- __Modify__ `src/Models/ToolbarOptions.ts` to allow Datagrids to accept incoming configs instead just overwriting with defaults.

- __Update__ `srcdocs/Basic Features.tsx` and `src/SampleFeatures.tsx` with the desired config (simple pagination, drop export/print button and search text).

### CODE REVIEW

File: srcdocs/components/SampleFeatures.tsx [hunk 1]
```diff
@@ -14,6 +14,13 @@ class SampleFeatures extends React.Component {
         errorMessage: null as any
     };
 
+    private toolbarOptions = {
```
Comment (AnyxLeo) [inline srcdocs/components/SampleFeatures.tsx]: revert this changes this grid must have the full features

File: srcdocs/components/SampleFeatures.tsx [hunk 2]
```diff
@@ -57,7 +64,7 @@ class SampleFeatures extends React.Component {
                                 </TableCell>
                             </TableRow>
                     }
-                    toolbarOptions={new ToolbarOptions()}
+                    toolbarOptions={new ToolbarOptions(this.toolbarOptions)}
```
Comment (AnyxLeo) [inline srcdocs/components/SampleFeatures.tsx]: Revert this

File: srcdocs/components/BasicFeatures.tsx [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
     withRemoteDataSource } from '../../src';
 
 import columns from '../utils/columns';
 
 const MyComponent = () => {
+    const toolbarOptions = {
+        advancePagination: false,
+        exportButton: false,
+        printButton: false,
+        searchText: false,
```
Comment (AnyxLeo) [inline srcdocs/components/BasicFeatures.tsx]: You are missing the top paginator

File: src/Models/ToolbarOptions.ts [hunk 4]
```diff
@@ -18,5 +18,11 @@ export default class ToolbarOptions {
        this.topPager = true;
        this.rowsPerPageOptions = [10, 20, 50, 100];
        this.itemsPerPage = 10;
+
+       if (options){
```
Comment (AnyxLeo) [inline src/Models/ToolbarOptions.ts]: This change failed for the others grid cases. This change set all the options as false when the options aren't declared it is wrong.

File: src/Models/ToolbarOptions.ts [hunk 5]
```diff
@@ -10,13 +10,13 @@ export default class ToolbarOptions {
...
        // TODO: Apply options
-       this.advancePagination = true;
-       this.bottomPager = true;
-       this.exportButton = true;
-       this.printButton = true;
-       this.searchText = true;
-       this.topPager = true;
-       this.rowsPerPageOptions = [10, 20, 50, 100];
-       this.itemsPerPage = 10;
+       this.advancePagination = options && 'advancePagination' in options? options['advancePagination'] : true;
```
Comment (geoperez) [inline src/Models/ToolbarOptions.ts]: Don't use the bracket notation.
